frontend/src/metabase-types/api/data-app.ts

```diff
@@ -45,14 +45,14 @@ export type ActionParametersMapping = Pick<
 
 export interface ActionDashboardCard
   extends Omit<BaseDashboardOrderedCard, "parameter_mappings"> {
-  action_id: number | null;
```

Comment:
we're no longer using action_id at the top level. We can tell if an action is implicit now by the `type: implicit` property